apps/charterafrica/migrations/20240111_075951_update_blocks_rename_collection_to_items.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrateUpArgs, MigrateDownArgs } from "@payloadcms/db-mongodb";
```

Comment:
Isn't this the old migrations? It should run before the v1_v2 migration, right?

Reply:
Recreated payload migrations to ensure they run after old migrations